services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -279,6 +284,48 @@ public void waypointTargetsList() {
     assertEquals(";;5.55,7.77;1.22,3.44;;1.55,8.99", routeOptions.waypointTargets());
   }
 
+  @Test
+  public void annotationsString() {
+    String annotationsStr = ";" + ANNOTATION_MAXSPEED + ";" + ";" + ANNOTATION_DURATION + ";";
```

Comment:
This is not valid for annotations:

> InvalidInput
>
> annotations value must be one of duration, distance, speed, congestion

`congestion;duration` ✅ 
`;congestion;duration;` ❌ 

Annotations only accepts a list of valid `DirectionsCriteria` annotations separated by `;`

The same thing applies to 👇

Reply:
fixed